components/fonts/baekmuk/history

```diff
@@ -1 +1,2 @@
-system/font/baekmuk-ttf@2.2-2018.0.0.1 system/font/baekmuk
+system/font/baekmuk-ttf@2.2-2018.0.0.1 system/font/truetype/baekmuk
+system/font/baekmuk@2.2-2018.0.0.2 system/font/truetype/baekmuk
```

Comment:
Shouldn't it be both `2018.0.0.2` given that we want to replace `2018.0.0.1` and older?
```
$ pkg info -r system/font/baekmuk-ttf system/font/baekmuk
          FMRI: pkg://openindiana.org/system/font/baekmuk@2.2-2018.0.0.1:20181106T081715Z
          FMRI: pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.1:20190720T075302Z
```

Reply:
Probably yes... I forgot that they had been republished... not sure what happens when there is a rename to a fmri that does not exist... because in the previous revision only system/font/truetype/baekmuk@2.2-2018.0.0.1 was actually published and actually provided content... better bump.

Reply:
Hmm, I don't know... Maybe we should wait for @pyhalov.